### ISSUE TITLE
Results by event?

### ISSUE THREAD

**Issue by lperovskaya** [NEERC/volunteers#68]
ITMO volunteer system requires to push results for each separate event. Is there a way to reasonably evaluate volunteers per event?

**Comment by matvey-kazakov** [comment]
You mean do not sum with previous year results? Sure it is possible. It will be just sorted by evaluation list. We just had no need in this.

**Comment by lperovskaya** [comment]
No, I mean each year we now only have one assessment, but itmo needs four each year - one per event (QF + SPbPC + NEERC + ORSPC)
